frontend/js/actions/featuregrid.js

```diff
@@ -84,12 +88,54 @@ function selectAllQgis(featureTypeName, filterObj, ogcVersion, params, wfsUrl) {
     };
 }
 
+const selectMLS = (layers) =>{
+    return {
+        type: SELECT_MLS,
+        layers
+    };
+};
+
+const configureMultiLayerSelection = (filterObj = {}, zoomEnabled = false) => {
+    return (dispatch, getState) => {
+        const {siradec, map, layers: msLayers} = getState();
+        const crs = CoordinatesUtils.normalizeSRS(map?.present?.projection);
+        const activeFeatureType = siradec?.activeFeatureType || '';
+        const {multiLayerSelect = [], featureTypeName = '', layer = {}} = siradec?.configOggetti?.[activeFeatureType];
+        const layers = multiLayerSelect.map(({name, title = ''}) => {
+            const SLD_BODY = FilterUtils.getSLD(featureTypeName, filterObj, "1.0.0", "ogc", {}, name);
```

Comment:
the filter to use for getSLD depends on the configuration to add above:

```
"/path/attribute" = <value>
```

Where the value to use should be argument of configureMultiLayerSelection, instead of filterObj:

```
const configureMultiLayerSelection = (filterValue, zoomEnabled = false) => {
....
const layers = multiLayerSelect.map(({name, title = '', filterOn}) => {
   const filterObj = <create filter for filterOn + "=" + value >
   const SLD_BODY = FilterUtils.getSLD(featureTypeName, , filterObj, "1.0.0", "ogc", {}, name);
....
}
```